### PR TITLE
Fix/chat-1066-01: Resolve some bugs in chat

### PR DESCRIPTION
**Pull request by hassenelkamel-exo**
- fix the problem of title " Chat " in tab.
- fix the problem of cannot read property
taskId= 23392

### CODE REVIEW

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue [hunk 1]
```diff
@@ -49,36 +50,43 @@ export default {
...
       showChatDrawer:false,
       listOfContact: false,
       fullNameOfUser:'',
       isOnline : true,
-      searchTerm:''
+      searchTerm:'',
+      conversationArea: false,
+      loggedout: false,
+      connected: false,
+      participantsArea: false,
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:65]: Unused variables, they should be removed.

Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:65]: some variables used in created(); conversationArea used in setSelectedContact, so if i remove them , i will remove from the other palce in code

Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:65]: i keep only loggedout, connected. because it store the status of user

Comment (ahamdi) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:65]: I didn't find a usage for those variables in the current PR, are they used already ?

Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:65]: in created()  call this in usrLoggedout() and connectionEstablished then for exp: in `connectionEstablished() `it update the status` this.setStatus(this.userSettings.originalStatus);`

Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:65]: For the var loggedout, we use it to change user status to offline `changeUserStatusToOffline`

Comment (ahamdi) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:65]: Ok, then those variables were missing from the ChatDrawer ?

Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:65]: ok i will remove this variables, after testing we dont need thoe

**Pull request merged**